### ISSUE TITLE
Accessibility - Focus Indicator Color Contrast

### ISSUE THREAD

**Issue by burgwyn** [18F/State-TalentMAP#1625]
## Focus Indicator is barely visible when focused on a number of elements throughout the web page.          

Users with visual impairments, such as color-blindness,   will have a difficult time following the Focus indicator. Screenshots   provided show Color Contrast failure with the focus indicator on actionable   elements.

## 2.4.7   Focus Visible (Level AA)
Any keyboard operable user interface has a mode of operation where the   keyboard focus indicator is visible.  Recommendation: Ensure   that it is visually apparent which page element has the current keyboard   focus: i.e., as you tab through the page, you can see where you are.          Link: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html
<img width="635" alt="screen shot 2018-03-18 at 9 12 39 pm" src="https://user-images.githubusercontent.com/661204/37573696-1eedc802-2af1-11e8-99b5-a0dce5ea8021.png">
